src/plugins/gauge/GaugePlugin.js

```diff
@@ -145,11 +145,11 @@ export default function () {
                             limitHigh: true
                         };
 
-                        if (min === '') {
+                        if (min === '' || min < 0) {
```

Comment:
I have just sow image of the gauges and they have negative minimum value, so to fix this just delete `min < 0` to fix that problem.